### DISCUSSION TITLE
offline mode configurating

### DISCUSSION THREAD

**Discussion by nochinator** [redeclipse/base#1400] [Help and Support] [answered]
I've search the docs at redeclipse.com and could not find anything about configuring offline mode game-play. is it possible to do this, and if so where can I find some docs on what can be configured and how to do so.

**Comment by dc-redeclipse** [comment, marked as answer]
Hey and welcome,
its called "Practice Match" in 2.0 and is basicly playing offline vs bots.
Is that what you are looking for?

**Reply by nochinator** [reply]
I have found that, but I am wondering about a way to configure the bots in that mode. especially the number of bots and there difficulty, sorry for the confusion

**Reply by Jigoku** [reply]
You can add and remove bots by pressing Insert or Delete. Or alternatively running the command `/botoffset N` where N is a number value for the total bots in the match.

**Reply by Calinou** [reply]
Bot skill is set with the `/botskillmin N` and `/botskillmax N` commands, where `N` is the skill level between 1 (very bad) and 101 (inhumanly good). Typical skill levels that would match online players are between 70 and 90. Skill levels are chosen randomly between `min` and `max`.

**Reply by nochinator** [reply]
thanks guys, where can I find a wiki and/or docs on this kind of stuff, I couldn't find it in the docs at redeclipse.net. if there isn't documentation on this I'd be more than happy to make them

**Comment by MirceaKitsune** [comment]
I believe this is a feature suggestion, one I brought up myself a while back. At the moment the Offline Match menu doesn't let you configure the standard options most FPS's offer. I understand this is in part related to persisting the right server settings. I too believe we should have settings for at least the following basics:

- Frag limit: Unlimited when 0 (unlimited default).
- Time limit: Unlimited when 0 (10 min default).
- Bot number: When 0 (default) use the map's recommended player count.
- Bot difficulty: The actual setting is a range between a min / max skill so the value should probably be offset -10 / +10.

**Reply by nochinator** [reply]
ok, well I look forward to when that is implemented
